dspace/config/spring/api/external-services.xml

```diff
@@ -92,5 +94,38 @@
         </property>
     </bean>
 
+    <bean id="scopusLiveImportDataProvider" class="org.dspace.external.provider.impl.LiveImportDataProvider">
+        <property name="metadataSource" ref="ScopusImportService"/>
+        <property name="sourceIdentifier" value="scopus"/>
+        <property name="recordIdMetadata" value="dc.identifier.scopus"/>
+        <property name="supportedEntityTypes">
+            <list>
+                <value>Publication</value>
+            </list>
+        </property>
+    </bean>
+
+    <bean id="pubmedEULiveImportDataProvider" class="org.dspace.external.provider.impl.LiveImportDataProvider">
+        <property name="metadataSource" ref="PubmedEuropeImportService"/>
+        <property name="sourceIdentifier" value="pubmedeu"/>
+        <property name="recordIdMetadata" value="dc.identifier.pmid"/>
```

Comment:
Again, we don't have this field in DSpace.  Should we add a `dc.identifier.pmid`?

Reply:
Feedback here wasn't addressed. I don't think this metadata field exists in DSpace out-of-the-box.